packages/widget/src/hooks/useGetTokenBalancesWithRetry.ts

```diff
@@ -14,7 +14,7 @@ export const useGetTokenBalancesWithRetry = (provider?: Provider) => {
       depth = 0,
     ): Promise<TokenAmount[] | undefined> => {
       try {
-        const walletAddress = isAddress(accountAddress)
+        const walletAddress = isLiFiAddress(accountAddress)
```

Comment:
I'm not sure whether `lifi.getTokenBalance` will fail if it receives a non-EVM address. Probably will, so it will have to be a separate task.